api/user.py

```diff
@@ -1,7 +1,14 @@
 from flask_login import UserMixin
 
+from enum import IntEnum
+
 import faf.db as db
 
+class UserGroup(IntEnum):
+    NONE = 0,
+    MODERATOR = 1,
+    ADMIN = 2
```

Comment:
As I have seen, you use operators like >= on the enum.
I'd suggest leaving some space between the numbers, so we could introduce more intermediate sublevels later (supermoderators, special authenticated user)
Maybe NONE = 0, MODERATOR = 10, ADMIN = 20